### PR TITLE
[4.0] Array to php 5.4+ notation [Unit Tests 1: core folder]

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Improvement.

### Summary of Changes

Since 4.0 doesn't support php 5.3 anymore, use php 5.4+ array notation in unit tests core folder.

### Testing Instructions

Simple code review.
Unit tests (travis) test passed.

### Documentation Changes Required

None.

### CODE REVIEW

File: tests/unit/core/mock/menu.php [hunk 1]
```diff
@@ -82,28 +82,28 @@ public static function create(PHPUnit_Framework_TestCase $test, $setDefault = tr
...
+		$return   = [];
+		$return[] = ['en-GB', self::$data[45]];
 
 		return $return;
 	}
 
 	public static function prepareGetItemsData($attributes, $values)
 	{
-		$items = array();
+		$items = [];
```
Comment (Quy) [inline tests/unit/core/mock/menu.php]: Align

File: tests/unit/core/mock/input.php [hunk 2]
```diff
@@ -37,7 +37,7 @@ class TestMockInput
 	 */
 	public function __construct(PHPUnit_Framework_TestCase $test)
 	{
-		self::$inputs = array();
+		self::$inputs = [];
```
Comment (Quy) [inline tests/unit/core/mock/input.php]: Align

File: tests/unit/core/mock/menu.php (hunk 1 above)
Comment (andrepereiradasilva) [inline tests/unit/core/mock/menu.php]: all aligned

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5b3056d31a4b0318cbad06e4daeff7ff7abb8d95<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16657">issues.joomla.org/tracker/joomla-cms/16657</a>.</sub>

**Pull request closed without merging**